.github/workflows/spelling.yml

```diff
@@ -98,34 +98,39 @@ jobs:
         use_sarif: ${{ (!github.event.pull_request || (github.event.pull_request.head.repo.full_name == github.repository)) && 1 }}
         extra_dictionary_limit: 20
         extra_dictionaries:
```

Comment:
v0.0.22 reorganized the cspell dictionaries signficantly as mentioned on https://github.com/check-spelling/check-spelling/releases/tag/v0.0.22 . I updated the list by removing all of them and looking at the suggestions from the `check-spell` action.